Source/WebCore/html/canvas/CanvasRenderingContext2DBase.h

```diff
@@ -304,7 +308,6 @@ class CanvasRenderingContext2DBase : public CanvasRenderingContext, public Canva
 
         String unparsedFont;
         FontProxy font;
```

Comment:
recipe for disaster to be missing ref-counting and non-refcounting for the same object. Someone would ref this object and think they're keeping it alive but then this could get destroyed anyway.

Note that this not heap-allocated. If we switch to a Ref / RefPtr here, then we'll start making heap allocations in cases where we previously didn't, which could have performance implications.

Reply:
Switched to `RefPtr`.